rlbench/backend/scene.py

```diff
@@ -369,6 +369,7 @@ def get_demo(self, record: bool = True,
                 done = False
                 success = False
                 while not done:
+                    gripper_open = 1.0
```

Comment:
I believe this should be moved below line 340 instead. Otherwise, you will overwrite the value that gripper_open has when `record=True`